### ISSUE TITLE
MS-Celeb vs CASIA model 

### ISSUE THREAD

**Issue by aleksandar** [davidsandberg/facenet#401]
Hi @davidsandberg ,
do you maybe measured results on FAR 10e-4? There are some interesting results. 
Model trained on CASIA dataset is better on this FAR (10e-4) than model trained on MS-celeb.
Test protocol is mix of LFW and blufr (http://www.cbsr.ia.ac.cn/users/scliao/projects/blufr/) protocols.
Your models was tested and also models trained on these sets (following your best practices).

You can "see" this difference between models just by comparing histograms of distances on test set.
![casia hist](https://user-images.githubusercontent.com/229280/28714050-e5468cfe-7391-11e7-9962-0abbf658e175.png)
![ms-celeb hist](https://user-images.githubusercontent.com/229280/28714078-0f8eff46-7392-11e7-8cd9-0a93f9f9bbce.png)

**Comment by qixianbiao** [comment]
Have not done something like this. Remind you that the features should be L2 normed before computing distance or similarity....

**Comment by aleksandar** [comment]
the embeddings layer is normalized by default in david's implementation. 
the question is why these two models have a little bit different distributions, acutally variances on same test pairs. model trained on casia set has lower variance and this is better for lower FAR.

**Comment by Shahnawazgrewal** [comment]
@aleksandar did you clean MS-Celeb dataset before training?

**Comment by aleksandar** [comment]
yes, we did. chinese whispers algorithm was used, and model trained on casia set.

**Comment by Shahnawazgrewal** [comment]
I developed a pipeline to clean the MS-Celeb dataset however I am not sure if I can publish this work. Any suggestions? @aleksandar

**Comment by aleksandar** [comment]
DBSCAN is also good algorithm for this, nice idea. regarding permissions on this project you should ask the owner, @davidsandberg
